### PR TITLE
Feat/3.2 UI providers

### PR DESCRIPTION
**Pull request by foodaka**

### CODE REVIEW

File: scripts/configs/pools/ethereum.ts [hunk 1]
```diff
@@ -94,8 +94,8 @@ export const lidoEthereumMainnetProtoV3Pool: PoolConfig = {
     REPAY_WITH_COLLATERAL_ADAPTER: '0x66E1aBdb06e7363a618D65a910c540dfED23754f',
     SWAP_COLLATERAL_ADAPTER: '0xD0887AA7fEBC8962c622493646195e7c76D94fCE',
     WITHDRAW_SWAP_ADAPTER: '0x43eDB797834151D041619EEF833Edc784B509dAE',
-    UI_INCENTIVE_DATA_PROVIDER: '0xB3a0fA312853A8E24C7756e6857cD56592b24c99',
-    UI_POOL_DATA_PROVIDER: '0x07060168141ab016abb8933b86d2541d66b4f74a',
+    UI_INCENTIVE_DATA_PROVIDER: '0x162A7AC02f547ad796CA549f757e2b8d1D9b10a6',
```
Comment (grothem) [inline scripts/configs/pools/ethereum.ts]:
```suggestion
    UI_INCENTIVE_DATA_PROVIDER: '0xe3dff4052f0bf6134acb73beae8fe2317d71f047',
```

File: scripts/configs/pools/ethereum.ts [hunk 2]
```diff
@@ -94,8 +94,8 @@ export const lidoEthereumMainnetProtoV3Pool: PoolConfig = {
...
+    UI_POOL_DATA_PROVIDER: '0xe3dff4052f0bf6134acb73beae8fe2317d71f047',
```
Comment (grothem) [inline scripts/configs/pools/ethereum.ts]:
```suggestion
    UI_POOL_DATA_PROVIDER: '0x3f78bbd206e4d3c504eb854232eda7e47e9fd8fc',
```

Comment (foodaka) [reply scripts/configs/pools/ethereum.ts]: updated in 4a34643596ecd15981074fd08711b7c462f08953

File: scripts/configs/pools/ethereum.ts (hunk 1 above)
Comment (foodaka) [reply scripts/configs/pools/ethereum.ts]: updated in 4a34643596ecd15981074fd08711b7c462f08953

**Review by grothem — APPROVED** [review]

**Pull request merged**